temporal-sdk/src/main/java/io/temporal/internal/common/CheckedExceptionWrapper.java

```diff
@@ -69,25 +69,11 @@ public static RuntimeException wrap(Throwable e) {
   }
 
   /**
-   * Removes CheckedException wrapper from the whole chain of Exceptions. Assumes that wrapper
+   * Removes CheckedExceptionWrapper from the top of the chain of Exceptions. Assumes that wrapper
```

Comment:
Remove causeField and the related initialization code above.
Remove setThrowableCause method below.

Reply:
Done